core/includes/drupal.inc

```diff
@@ -2591,6 +2591,51 @@ function views_clean_css_identifier($identifier, $filter = array(' ' => '-', '/'
   backdrop_clean_css_identifier($identifier, $filter);
 }
 
+/**
+ * Finds the most recently changed nodes that are available to the current user.
+ *
+ * @param (int) $number
+ *   (optional) The maximum number of nodes to find. Defaults to 10.
+ *
+ * @return array
+ *   An array of node entities or an empty array if there are no recent nodes
+ *   visible to the current user.
+ */
+function node_get_recent($limit = 10) {
+  $query = db_select('node', 'n');
+
+  if (!user_access('bypass node access') && !user_access('view any unpublished content')) {
+    // If the user is able to view unpublished nodes, allow them to see these
+    // in addition to published nodes. Check that they actually have some
+    // unpublished nodes to view before adding the condition.
+    // Note: this query still ignores unpublished by type permissions.
+    // See: https://github.com/backdrop/backdrop-issues/issues/146.
+    $own_unpublished = db_query('SELECT nid FROM {node} WHERE uid = :uid AND status = :status',
+    array(':uid' => $GLOBALS['user']->uid, ':status' => NODE_NOT_PUBLISHED))->fetchCol();
```

Comment:
This is not a separate line, but part of the parameters being passed to the `db_query()` in the line above. It needs to be indented by 2 spaces:
```suggestion
      array(':uid' => $GLOBALS['user']->uid, ':status' => NODE_NOT_PUBLISHED))->fetchCol();
```

Reply:
Done